docs/credits.md

```diff
@@ -1353,19 +1338,17 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | patch-package                                     | David Sheldrick                                                           | [https://github.com/ds300/patch-package](https://github.com/ds300/patch-package)                                                                                                                                            | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | path-case                                         | Blake Embrey                                                              | [https://github.com/blakeembrey/change-case](https://github.com/blakeembrey/change-case)                                                                                                                                    | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
-| path-dirname                                      | Elan Shanker                                                              | [https://github.com/es128/path-dirname](https://github.com/es128/path-dirname)                                                                                                                                              | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | path-exists                                       | Sindre Sorhus                                                             | [https://github.com/sindresorhus/path-exists](https://github.com/sindresorhus/path-exists)                                                                                                                                  | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | path-is-absolute                                  | Sindre Sorhus                                                             | [https://github.com/sindresorhus/path-is-absolute](https://github.com/sindresorhus/path-is-absolute)                                                                                                                        | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | path-is-inside                                    | Domenic Denicola                                                          | [https://github.com/domenic/path-is-inside](https://github.com/domenic/path-is-inside)                                                                                                                                      | [WTFPL](https://spdx.org/licenses/WTFPL.html)                                                                      [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
```

Comment:
@michaelnoonan WTFPL detected xD. I think this might be against policy https://octopushq.atlassian.net/wiki/spaces/RND/pages/3108765730/Using+Open+Source+in+Octopus+projects (change at line 1345)

Reply:
```
  is-path-inside@2.1.0:
    dependencies:
      path-is-inside: 1.0.2
```

Just Npm Things. It looks like it is ultimately a dependency of eslint, so isn't a production dependency.